workflow-service/src/main/java/com/redhat/parodos/workflow/execution/entity/AbstractEntity.java

```diff
@@ -49,7 +49,7 @@ public UUID getId() {
         return this.id;
     }
 
-    void setId(UUID id) {
+    public void setId(UUID id) {
```

Comment:
Why not to go with `protected`?

Reply:
This change is fine. Not sure why that scope was chosen